fcrepo-jms/src/test/java/org/fcrepo/integration/jms/observer/HeadersJMSIT.java

```diff
@@ -131,7 +134,8 @@ public void testIngestion() throws RepositoryException {
         try {
             containerService.findOrCreate(session, testIngested);
             session.save();
-            awaitMessageOrFail(testIngested, NODE_ADDED_EVENT_TYPE, null);
+            final Message ingestMsg = awaitMessageOrFail(testIngested, NODE_ADDED_EVENT_TYPE);
+            assertTrue(getResourceTypes(ingestMsg).contains("fedora:Resource"));
```

Comment:
The namespace-resolution is something that is a useless expense at the event bus.